msgpack.d.ts

```diff
@@ -1,16 +1,28 @@
 /**
  * Serializes a value to a MessagePack byte array.
- * @param data The value to serialize. This can be a scalar, array or object.
- * @param multiple Indicates whether multiple values in data are concatenated to multiple MessagePack arrays.
+ * @param data An object that defined additional options.
+ * @param options Indicates whether multiple values in data are concatenated to multiple MessagePack arrays.
  */
-export function serialize(data: any, multiple: boolean = false): Uint8Array;
+export function serialize(
+    data: any,
+    options: {
+        /** Indicates whether multiple values in data are concatenated to multiple MessagePack arrays. */
+        multiple?: boolean,

```

Comment:
Is there a way to describe the default value that applies if nothing is specified? Like it was with the previous `boolean` parameter that defaulted to `false`. I always find it helpful to know what happens when I'm not setting any value.

Reply:
Your addition of `= false` was illegal actually. TypeScript only allows 'default values' to be specified in method *implementations*, not declarations, so your `= false` was producing a typescript compiler error.
The only resort we have is to document the default behavior. I've added docs to that effect.